### DISCUSSION TITLE
An unexpected internal error occurred (Vercel build)

### DISCUSSION THREAD

**Discussion by verfasor** [vercel/community#1417] [Help] [answered]
### Update

According to https://www.vercel-status.com/,
```
Identified - A service degradation concerning builds and deployments has been identified and a fix is being implemented.
Feb 01, 2023 - 16:17 UTC
```

### Summary

It's a Next.js App and I only made minor changes to the code-base.
It throws the following error resulting in a build error.

```
[00:14:31.803] Collected static files (public/, static/, .next/static): 7.929ms
[00:14:31.904] Build Completed in /vercel/output [1m]
[00:14:33.400] An unexpected internal error occurred
```

I tried deploying the last successful build, but it ended up with the same error.
What seem to be the problem?

**Comment by swarnava** [comment, marked as answer]
Hi,

This should be fixed now: https://www.vercel-status.com/incidents/4gyl0x5r8wgh
